code/modules/mob/living/carbon/human/species/species.dm

```diff
@@ -189,6 +192,70 @@
 			SPAN_NOTICE("You pat [target] on the back to make [t_him] feel better!"), null, 4)
 	playsound(target, 'sound/weapons/thudswoosh.ogg', 25, 1, 5)
 
+/datum/species/proc/attempt_rock_paper_scissors(var/mob/living/carbon/human/H, var/mob/living/carbon/human/target)
+	if(!H.get_limb("r_hand") && !H.get_limb("l_hand"))
+		to_chat(H, SPAN_NOTICE("You have no hands!"))
+		return
+
+	if(!target.get_limb("r_hand") && !target.get_limb("l_hand"))
+		to_chat(H, SPAN_NOTICE("They have no hands!"))
+		return
+
+	//Responding to a raised hand
+	if(target.flags_emote & EMOTING_ROCK_PAPER_SCISSORS && do_after(H, 5, INTERRUPT_MOVED, EMOTE_ICON_ROCK_PAPER_SCISSORS))
+		if(!(target.flags_emote & EMOTING_ROCK_PAPER_SCISSORS)) //Additional check for if the target moved or was already high fived.
+			to_chat(H, SPAN_NOTICE("Too slow!"))
+			return
+		target.flags_emote &= ~EMOTING_ROCK_PAPER_SCISSORS
+		var/static/list/game_quips = list("Rock...", "Paper...", "Scissors...", "Shoot!")
+		for(var/quip in game_quips)
+			if(!H.Adjacent(target))
+				to_chat(list(H, target), SPAN_WARNING("You need to be adjacent to eachother to play!"))
+				return
+			to_chat(list(H, target), SPAN_NOTICE(quip))
+			sleep(5)
+		var/static/list/intent_to_play = list(
+			"[INTENT_HELP]" = "random",
```

Comment:
make random have a 1% chance for "gun" which beats everything it will be funny

Reply:
sounds like an expansion pack i can release later for $15